### ISSUE TITLE
Credits modal

### ISSUE THREAD

**Issue by sovesove** [MPH-Bali/green-village-project#64]
On the footer of the public facing pages there will be a link "Credits". 

We can potentially have a full page with all the persons involved (and their roles?)

Also, we NEED to add credits for the free license icons we've used in the design.
See Trello card for the details:
https://trello.com/c/WTBQJmY3/41-icons-designers-credits

**Comment by sovesove** [comment]
Team contributor credits:
https://trello.com/c/mhtpgCTN/68-team-member-credits
Currently empty - TBC

**Comment by nicksarafa** [comment]
@sovesove do we have a design on this section? we have to communicate open source licenses too which usually is just a plain text file saved to seperate uri

**Comment by sovesove** [comment]
Yes, see the trello cards. They should be merged actually but there is an issue with the "member credits" that we need to discuss, so maybe keep "18.0 Team member Credits" on hold.
